js/react_native/ios/OnnxruntimeModuleTest/OnnxruntimeModuleTest.mm

```diff
@@ -16,15 +16,17 @@ @implementation OnnxruntimeModuleTest
 - (void)testOnnxruntimeModule {
   NSBundle *bundle = [NSBundle bundleForClass:[OnnxruntimeModuleTest class]];
   NSString *dataPath = [bundle pathForResource:@"test_types_float" ofType:@"ort"];
+  NSString *sessionKey = @"";
 
   OnnxruntimeModule *onnxruntimeModule = [OnnxruntimeModule new];
 
-  // test loadModel()
+  // test loadModelFromBuffer()
   {
     NSMutableDictionary *options = [NSMutableDictionary dictionary];
-    NSDictionary *resultMap = [onnxruntimeModule loadModel:dataPath options:options];
+    NSData *fileData = [NSData dataWithContentsOfFile: dataPath];
+    NSDictionary *resultMap = [onnxruntimeModule loadModelFromBuffer:fileData options:options];
```

Comment:
Is there another test that exercises the loadModel path so we cover that and loadModelFromBuffer?